package.json

```diff
@@ -41,7 +41,7 @@
   },
   "dependencies": {
     "@primer/octicons": "^9.1.1",
-    "@primer/primitives": "^4.0.1"
+    "@primer/primitives": "^4.0.2"
```

Comment:
@jonrohan Do we even need to do this? With the `^` it should automatically install the latest version as long as it's not a major version with breaking changes.

Reply:
Maybe we should start saving exact versions like dotcom